src/content/resources/cpp/multidimensional-arrays.md

```diff
@@ -0,0 +1,122 @@
+---
+authors:
+  - "alearori#8661"
+created_at: 2019/11/15
+title: "Multidimensional Arrays"
+---
+
+## What is a multidimensional array?
+
+To put simply, a multidimensional array is an array that contains another array of another array... of another array of objects. It is commonly used for making matricies, but it can also be used for storing information such as if you use `string`. 
+
+The format of the container looks like: 
+
+`{{a1,a2,a3,...},{b1,b2,b3,...},{c1,c2,c3,...},...}`
+
+Where `a1`,`a2`,`a3`, and etc. are objects of the data type that was array was declared to contain at the start.
+But they can also be another array whch contains objects of said data type. Which looks like this:
+
+`{{{a1,a2,a3,...},{b1,b2,b3,...},{c1,c2,c3,...},...},...}`
+
+And we can keep going on and on, creating more `{}` inside the `{}`.
+
+### It is NOT an array inside of an array
```

Comment:
This title doesn't make sense anymore. We've previously stated that "a multidimensional array is an array that contains another array of another array".

Reply:
Okay, I moved that portion into Restrictions, and deleted that header.